code/parse/parselo.cpp

```diff
@@ -450,8 +450,8 @@ int required_string(const char *pstr)
 
 	if (count == RS_MAX_TRIES) {
 		nprintf(("Error", "Error: Unable to find required token [%s]\n", pstr));
```

Comment:
`error_display()` already does this, so this line is redundant and can be removed.

Reply:
Fixed.